bindings/wasm/examples/gltf-io.js

```diff
@@ -144,6 +155,28 @@ export function writeMesh(doc, manifoldMesh, attributes, materials) {
   const buffer = doc.getRoot().listBuffers()[0];
   const manifoldExtension = doc.createExtension(EXTManifold);
 
+  const attributeUnion = [];
+  for (const matAttributes of attributes) {
+    matAttributes.forEach((attribute, i) => {
```

Comment:
The input `attributes` is now a `string[][]` instead of `string[]`, since different materials can have different attributes.